src/pages/Location.jsx

```diff
@@ -8,9 +7,9 @@ import { Zipcode, ZipData } from '../components/Location';
 import { FormDataContext } from '../context/FormDataContext';
 import { validateZipCode } from '../helpers/api';
 import '../styles/page.css';
-
+import { useSelector } from 'react-redux';
 const Location = (props) => {
-  const { formData } = useContext(FormDataContext);
+  const  formData  = useSelector(state=>state.formdatacontext);
```

Comment:
So theres a bug within the Location/zipcode state. It's getting returned as a `Promise`. The fix should be in here or in one of it's nested objects where it's calling. Probably just need to add an `await` somewhere.